gulpfile.babel.js

```diff
@@ -0,0 +1,48 @@
+import gulp from 'gulp';
+import jasmineNode from 'gulp-jasmine-node';
+import gls from 'gulp-live-server';
+import babel from 'gulp-babel';
+import injectModules from 'gulp-inject-modules';
+import gulpBabelIstanbul from 'gulp-babel-istanbul';
+import gulpCoveralls from 'gulp-coveralls';
+import env from 'gulp-env';
+import exit from 'gulp-exit';
+
+require('dotenv').config();
+
```

Comment:
Single line comment to describe what each gulp task does like that of coverage will be much better